modules/llm-cache/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ file(GLOB VINEYARD_LLM_CACHE_SRCS "${CMAKE_CURRENT_SOURCE_DIR}"
 )
 
 add_library(vineyard_llm_cache ${VINEYARD_LLM_CACHE_SRCS})
-target_link_libraries(vineyard_llm_cache PUBLIC vineyard_client vineyard_basic)
+target_link_libraries(vineyard_llm_cache PUBLIC libzstd_static ${GLOG_LIBRARIES} vineyard_client vineyard_basic)
```

Comment:
Don't publicly link aginist `libzstd_static`. As this target is not installed and exported. Use

```
target_link_libraries(... PRIVATE ...)
```

Reply:
Done